accounts/views.py

```diff
@@ -803,6 +805,18 @@ def handle_uploaded_file(user_id, f):
     return True
 
 
+def copy_sound_file_to_mirror_location(source_path):
```

Comment:
Can we put both copy functions in the same place in a util-y file? (so we don't have one in a view and one in a management command)